docs/get-started/create-a-project.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_label: 'Create a project'
+sidebar_position: 2
+---
+
+# Create a project
+
+First, create a project, move to it and initialize Git.
+```shell
+$ git init
```

Comment:
I'll do that, to be honest I think it decreases the readability of the tutorial as a whole but I guess that for some commands it would ease a user experience.

Reply:
Ok thanks for the resources. Let's keep it that way then.